### PR TITLE
fixes #3937 - correct title wording Editing to Edit

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

File: app/views/compute_resources/edit.html.erb [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<% title _("Editing compute") %>
+<% title _("Edit Compute resource") %>
```
Comment (GregSutcliffe) [inline app/views/compute_resources/edit.html.erb]: Shouldn't this be `"Edit %s", @compute_resource` ?

**Comment by GregSutcliffe** [comment]
:+1: after a small nitpick

File: app/views/compute_resources/edit.html.erb (hunk 1 above)
Comment (isratrade) [inline app/views/compute_resources/edit.html.erb]: @GregSutcliffe, now that we use two-pane, we rarely see the title for the edit page.  I checked the code and we use `Edit %s` for only hosts, hostgroups, images, and taxonomies.  All the others, the title is static "Edit Domain", etc.  Any, I updated the PR to use Edit %s`  on compute resources

**Comment by domcleal** [comment]
Thanks @isratrade, merged as 9b5fe2c.

**Pull request closed without merging**